core_lib/data/resources/kb.ini

```diff
@@ -54,6 +54,10 @@ CmdMoveFrameForward=Ctrl+.
 CmdMoveFrameBackward=ctrl+","
 CmdAddFrame=F7
 CmdDuplicateFrame=F6
+CmdIncreaseFrameExposure=-
```

Comment:
We can change it to something else then, I have no stance really. I didn't think long and hard, I just picked one that wasn't bound already because It the shortcut was never my focus.

Reply:
Alright. I’d say let’s go with + for increase and - for decrease then, easy to remember.